src/systemtest/resources/sinceBeginningDateRange/expected/reposense_testrepo-Beta_master/authorship.json

```diff
@@ -2,6 +2,7 @@
   {
     "path": "src/main/resources/view/InfoPanel.fxml",
     "fileType": "fxml",
+    "isBinary": false,
```

Comment:
is it possible to output this field only if its true?

Reply:
It's possible. I'll just need to add a new type adapter [here](https://github.com/reposense/RepoSense/blob/0e39bb7ffa5094e13fb120a36e3c2666f2566572/src/main/java/reposense/util/FileUtil.java#L94). Should I do it?

Reply:
or another way is, we make it a default null field.
iirc, null object fields wont be outputted

Reply:
Thanks for the suggestion. Did just that.